lddd: add page (#9281)
tldr-pages/tldr: pages/linux/lddd.md

# lddd

> > Find broken library links on your system. This tool is only available on Archlinux.

Reviewer: We try to avoid `you`/`your`
```suggestion
> Find broken library links on the system.
> This tool is only available on Arch Linux.
```

> > More information: <https://man.archlinux.org/man/extra/devtools/lddd.1.en>.

Reviewer: ```suggestion
> More information: <https://man.archlinux.org/man/extra/devtools/lddd.1>.
```

> - Scan $PATH, /lib, /usr/lib, /usr/local/lib and /etc/ld.so.conf.d/*.conf to find out packages with broken library links:

Reviewer: I think we don't need to list the directories here, that's why we have the more information link
```suggestion
- Scan directories for packages with broken library links:
```

Reviewer 2: Agreed, Your description sounds better.
